src/Sarif.Multitool/Rules/RuleResources.resx

```diff
@@ -227,4 +227,10 @@ If the validator reports that 'contextRegion' is not a proper superset of 'regio
   <data name="SARIF2005_ProvideHelpfulToolInformation_Warning_UseNumericToolVersions_Text" xml:space="preserve">
     <value>{0}: Placeholder '{1}'</value>
   </data>
+  <data name="SARIF1004_ExpressUriBaseIdsCorrectly_Error_UriBaseIdValueMustNotContainDotDotSegment_Text" xml:space="preserve">
+    <value>{0}: '{1}' '{2}' Placeholder: SARIF1004_ExpressUriBaseIdsCorrectly_Error_UriBaseIdValueMustNotContainDotDotSegment_Text</value>
+  </data>
+  <data name="SARIF1004_ExpressUriBaseIdsCorrectly_Error_UriBaseIdValueMustNotContainQueryOrFragment_Text" xml:space="preserve">
+    <value>{0}: '{1}' '{2}' Placeholder: SARIF1004_ExpressUriBaseIdsCorrectly_Error_UriBaseIdValueMustNotContainQueryOrFragment_Text</value>
```

Comment:
>SARIF1004_ExpressUriBaseIdsCorrectly_Error_UriBaseIdValueMustNotContainQueryOrFragment_Text [](start = 41, length = 91)

I like this convention for the placeholders.

Reply:
thanks! will help code reviews later when we replace them :)

---
In reply to: [445142997](https://github.com/microsoft/sarif-sdk/pull/1930#discussion_r445142997) [](ancestors = 445142997)